CryptoPrices/Sources/Home/Sources/Home/Localization/Strings.swift

```diff
@@ -1,20 +1,61 @@
-//
-//  Strings.swift
-//  Home
-//
-//  Created by Minh Pham on 06/12/2022.
-//
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
 import Foundation
 
-public enum Strings {
+// swiftlint:disable superfluous_disable_command file_length implicit_return prefer_self_in_static_references
```

Comment:
Are all the `swiftlint: disable` generated by SwiftGen? As I see, we have `swiftlint:disable all` at the top file, I think the rest `swiftlint: disable`s are not needed.

Reply:
@nkhanh44  All this is generated by `swiftgen` by its defaul template. So while this is redundant we don't want to waste effort to edit the files that might be re-written in anytime 🙏

Reply:
Yeah, thank you. Swiftgen's template looks a bit messy 😄.